doc/conf.py

```diff
@@ -48,11 +49,13 @@
 # built documents.
 #
 # The short X.Y version.
-execfile(os.path.join(os.path.dirname(__file__), '../happybase/_version.py'))
-version = __version__
+
+version_path = join(dirname(__file__), '../happybase/_version.py')
+with open(version_path) as fp:
+    version = imp.load_source("_version", version_path, fp).__version__
```

Comment:
`imp.load_source()` is not public api in python 3, and the `imp` module is deprecated and pending removal...